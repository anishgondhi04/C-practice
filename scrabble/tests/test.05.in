5
4
0 0 H hello
0 0 V howdy
0 4 H yahoo
4 0 V oxoxo
0

