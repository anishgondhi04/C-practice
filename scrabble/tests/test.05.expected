hello
o...x
w...o
d...x
yahoo
